### ISSUE TITLE
Postgres Enums not double quoted when used

### ISSUE THREAD

**Issue by tomsjansons** [drizzle-team/drizzle-kit-mirror#80]
When using a postgres enum in table, the type is not double quoted creating a problem for enum names that contain upper case letters.

![image](https://github.com/drizzle-team/drizzle-kit-mirror/assets/1428917/3004dff9-2e3e-4d25-98c4-e7ed713cc32b)

After running `drizzle-kit generate:pg`, the resulting migration has not double quoted `somePgEnum` in the table field
![image](https://github.com/drizzle-team/drizzle-kit-mirror/assets/1428917/72941a7f-5546-48e0-880d-942654cd8545)

repro example https://stackblitz.com/edit/drizzle-postgres-enum?file=index.ts

**Comment by A0nurfitri** [comment]
I also noticed the same issues, when delcaring enum like this 

```ts
export const transactionStatusEnum = pgEnum("TransactionStatusEnum", [
  "PENDING",
  "FAILED",
  "SUCCESS",
])
```
after running the `generate:pg` command the migration created doesn't add double quotes to the enum type in SQL

```SQL
CREATE TABLE IF NOT EXISTS "Transaction" (
	"id" text PRIMARY KEY NOT NULL,
	"status" TransactionStatusEnum NOT NULL
);
```

 resulting to error 

```
PostgresError: type "transactionstatusenum" does not exist
```
when running the migration.

I either need to declare my enum using lowercase seperated by underscore like this `transaction_status_enum` or I need to edit the generated migration adding the double quote, manually

**Comment by AndriiSherman** [comment]
Just fixed it. Will include in next release, today

**Comment by AndriiSherman** [comment]
Fix released in 0.19.0

**Comment by andresgutgon** [comment]
Hi @AndriiSherman not working for me. More details here
https://github.com/drizzle-team/drizzle-orm/issues/646#issuecomment-2184981454

Any idea of what could be happening. Doesn't look is generating the enum type at all
